similar/misc/args.cpp

```diff
@@ -105,7 +105,7 @@ static void AppendIniArgs(const char *filename, Arglist &Args)
 		PHYSFSX_gets_line_t<1024> line;
 		while (Args.size() < MAX_ARGS && PHYSFSX_fgets(line, f))
 		{
-			const auto separator = " ";
+			const auto separator = " \t\n\v\f\r";
```

Comment:
I included those just because those are all considered "whitespace". I agree that " \t" should be enough though.